src/v/cluster/tm_stm.cc

```diff
@@ -656,22 +597,20 @@ ss::future<tm_stm::op_status> tm_stm::add_partitions(
         co_return tm_stm::op_status::unknown;
     }
 
-    if (!is_transaction_ga()) {
-        bool just_started = tx.partitions.size() == 0 && tx.groups.size() == 0;
+    bool just_started = tx.partitions.size() == 0 && tx.groups.size() == 0;
```

Comment:
shouldn't we get rid of the whole block of code since it checks for if (!is_transaction_ga()).. or am I misreading it.

Reply:
you are right, i missed this